api/openapi-spec/swagger.json

```diff
@@ -16616,6 +16616,11 @@
      "name"
     ],
     "properties": {
+     "acpiIndex": {
+      "description": "If specified, the ACPI index is used to provide network interface device naming, that is stable across changes in PCI addresses assigned to the device. This value is required to be unique across all devices and be between 1 and (16*1024-1).",
+      "type": "integer",
+      "format": "int32"
```

Comment:
This file is auto-generated.

Reply:
Ahh right, let me check if we can put the validation at golang code.

Reply:
I think that we usually do not validate at the API level (otherwise the spec would have looked messy), if there is a need to validate it is berried deep in the application (or at the webhook).

But as I answered previously ([here](https://github.com/kubevirt/kubevirt/pull/9116#issuecomment-1409169799)), I prefer not to add any validation and let libvirt do the job.

Reply:
Ok, so the VMI will fail with a libvirt error at the status right ?

Reply:
It depends on libvirt logic. If the value will cause it to fail creating the domain, it will indeed fail with the regular logs and behavior of a failing domain creation.
But it may also ignore the value silently, in which case the result will be that the ACPI index is not passed to the guest.

Reply:
fair enough for now